### PR TITLE
Migrate from Travis-CI to GitHub actions

### PR DESCRIPTION
**Pull request by allsopp**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-20.04
+          - ubuntu-22.04
+        perl:
+          - "5.10"
```
Comment (jjatria) [inline .github/workflows/test.yml:20]: We claim to support all the way back to 5.8. Any reason not to include it?

Comment (jjatria) [inline .github/workflows/test.yml:20]: We claim to support all the way back to 5.8. Any reason not to include it?

**Review by jjatria — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (allsopp) [reply .github/workflows/test.yml:20]: I originally wrote it `5.08` and the action died so I assumed it was unsupported. With `5.8` instead it's working, so we can continue supporting 5.8 (for any poor souls stuck on that version) :tada:.

**Review by jjatria — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+<div>
+    <a href="https://github.com/cv-library/UUID4-Tiny/actions/workflows/test.yml"><img src="https://github.com/cv-library/UUID4-Tiny/actions/workflows/test.yml/badge.svg"></a>
+</div>
```
Comment (JRaspass) [inline README.md]: No need for HTML, you can do this with native markdown:
```md
[![GitHub Actions](https://github.com/cv-library/UUID4-Tiny/actions/workflows/test.yml/badge.svg)](https://github.com/cv-library/UUID4-Tiny/actions/workflows/test.yml)
```

**Review by jjatria — APPROVED** [review]

**Review by JRaspass — APPROVED** [review]

**Pull request merged**